spec/models/post_mover_spec.rb

```diff
@@ -2754,6 +2754,68 @@ def create_topic_user(user, topic, opts = {})
         expect(event[:params][0]).to eq(post_2)
         expect(event[:params][1]).to eq(topic_1.id)
       end
+
+      context "with freeze_original option" do
```

Comment:
We already have this context. Could you move those tests to line 2879?

Reply:
Ah but this one is nested in the `with event trigger context` so I thought better to put here, what do you think?

Reply:
Oh ok, yeah now I see that, lgtm.

We could have triggers with freeze original.